index.html

```diff
@@ -10,30 +10,40 @@
 <body>
     <header class="toolbar">
         <h1 class="logo">My Logo</h1>
-        <nav class="hamburger"> <img src="images/hamburger.png" alt=""></nav>
+        <nav class="hamburger"><img src="images/hamburger.png" alt=""></nav>
+        <nav class="desk-header">
+            <ul class="desk-menu">
+                <li><a href="">Portfolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">About</a></li>
+            </ul>
+        </nav>
     </header>
 
     <section class="headline">
-        <h2 class="headline-title">I’m Billal Glad to see you!</h2>
+        <h2 class="headline-title">I’m Billal <br> Glad to see you!</h2>
         <p class="headline-description">I’m a software developer! 
             I can help you build a product ,
              feature or website Look through some of my work and experience!
               If you like what you see and have a project you need coded,
                don’t hestiate to contact me.</p>
         <p class="social">LET'S CONNECT</p>
         <ul class="social-icons">
-            <li><img src="images/twitter.png" alt=""></li>
-            <li><img src="images/linkedin.png" alt=""></li>
-            <li><img src="images/M.png" alt=""></li>
-            <li><img src="images/github.png" alt=""></li>
-            <li><img src="images/hand.png" alt=""></li>
+            <li><a href=""><img src="images/twitter.png" alt=""></a></li>
+            <li><a href=""><img src="images/linkedin.png" alt=""></a></li>
+            <li><a href=""><img src="images/M.png" alt=""></a></li>
+            <li><a href=""><img src="images/github.png" alt=""></a></li>
+            <li><a href=""><img src="images/hand.png" alt=""></a></li>
         </ul>       
     </section>
```

Comment:
[OPTIONAL] 👍🏽 
- I think you have have added a little too much spacing to the content inside your `<section class="headline">`. I'd like to kindly suggest that you reduce them a bit so that they match the figma template as much as possible. 🤗 

Here is what I can see compared to figma 👇🏽 

### Your design
![alexbbd](https://user-images.githubusercontent.com/99246359/200036202-8cb3cff5-6081-4872-aef2-f6b01b25609e.PNG)

### Figma
![alexd](https://user-images.githubusercontent.com/99246359/200036271-a435e857-e6dd-4e42-bf64-bfc44cc0aaec.PNG)

Would be cool if you take these suggestions into consideration 🌸

Reply:
@Favourezeugwa Alright thanks